srcpkgs/pari/template

```diff
@@ -1,24 +1,29 @@
 # Template file for 'pari'
 pkgname=pari
-version=2.13.1
+version=2.13.2
 revision=1
 build_style=configure
 build_helper=qemu
 configure_script=./Configure
-configure_args="--prefix=/usr"
+configure_args="--prefix=/usr $(vopt_if pthreads --mt=pthread)"
 make_build_target=all
 make_check_target=test-all
+make_install_target="install install-bin-sta install-lib-sta"
```

Comment:
It seems to be running a compiler in the install steps, probably because of `install-bin-sta` and `install-lib-sta`. Maybe add targets to `make_build_target` so everything happens in `do_build`?

Reply:
I tried, but they don't seem to have compile targets for that, only install targets. If I'm not mistaken it's not compiling anything, just linking  (it takes a long time because of -flto).

If this is really important, I think I can easily patch the makefile to add compile targets for the static binary and library.

Reply:
Not super important, but would be nice to achieve upstream, if you can :)

Since it's using a configure build style, we don't need to pass args to make and it has a smaller chance of breaking, but it would be nice to confine building to `do_build`.

From what I see in https://pari.math.u-bordeaux.fr/cgi-bin/gitweb.cgi?p=pari.git;a=blob;f=config/Makefile.SH;h=ca8bbf31a718c39abc6450c38aa176f6b157738f;hb=HEAD , you might be able to do `make_build_target="all lib-sta lib-dyn gp-sta"`.

Reply:
I had to patch the makefile so I don't to need to change `do_build()`. I will try to upstream that patch.